kubernetes/eks/README.md

```diff
@@ -0,0 +1,118 @@
+# Redpanda on EKS
+
+Amazon Elastic Kubernetes Service (EKS) enables us to run the Redpanda Kubernetes cluster in the AWS cloud. It manages the control plane for us, so we can concentrate on what the worker nodes are doing.
+
+### Accessing the Redpanda EKS dev cluster
+
+**Prerequisites**
+
+1. [Install `kubectl`.](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+2. [Install `helm`.](https://helm.sh/docs/intro/install/)
+
+3. [Install the `aws` command-line.](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+
+   You will need this in your `.bash_profile` or equivalent shell initialization file (put in the keys that were given to you when your AWS was set up):
+
+   ```
+   export AWS_ACCESS_KEY_ID=<My_Access_Key_Id>
+   export AWS_SECRET_ACCESS_KEY=<My_Secret_Access_Key>
+   ```
+
+4. Check that you have Vectorized AWS access. This command should return your AWS identity:
```

Comment:
Ahh, is this why our public key is published? Can this doc be made more generic?